aleph-client/src/lib.rs

```diff
@@ -122,6 +124,27 @@ pub fn send_xt(connection: &Connection, xt: String, xt_name: &'static str, tx_st
     );
 }
 
+pub fn try_send_xt(
+    connection: &Connection,
+    xt: String,
```

Comment:
You probably shouldn't change the original method as that would break other people's code. Unless that's used mostly internally in which case I think you can make a commit that fixes all of that - advantages of a monorepo ;P

Reply:
actually the target method `send_extrinsic` expects `String` anyway, so I'm not sure what do we gain here

Reply:
What do you understand by 'type-safety' here then? That when `xt` is only `String` then it can be just rubbish?

Reply:
Yes.

Reply:
used `UncheckedExtrinsicV4<T>` instead, as discussed offline